src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -57,7 +57,7 @@
             TextTrimming="CharacterEllipsis"
             Margin="0,-3,5,0"
             Content="{Binding Title}"
-            ToolTip="{Binding Title}"
+            ToolTip="View Pull Request details"
```

Comment:
@paladique There doesn't appear to be a `Resources.resx` file in the `GitHub.VisualStudio` project, which is where the `PullRequestListItem.xaml` file lives. I'm not sure where the command description really belongs. Should it be in the same assembly as the ViewModels (which controls the other text that appears on the view). Maybe @grokys has some opinions?